client/src/app/site/deployment-center/deployment-center-setup/step-configure/configure-vsts-build/linux-frameworks/linux-frameworks.component.ts

```diff
@@ -92,13 +93,6 @@ export class LinuxFramworksComponent implements OnDestroy {
         };
       });
 
-      this.phpFrameworkVersions = phpStack.properties.majorVersions.map(x => {
```

Comment:
Was this piece repeated? Or I am seeing something wrong?

Reply:
It was repeated, the original is present from line 88-93